src/EnergyPlus/Furnaces.cc

```diff
@@ -10639,7 +10639,7 @@ namespace Furnaces {
             }
 
             if (Furnace(FurnaceNum).FurnaceType_Num != UnitarySys_HeatCool) {
-                if (QZnReq > SmallLoad) {
+                if ((QZnReq > SmallLoad) && HeatingLoad) {
```

Comment:
When there is a dehumidification load, `HeatingLOad` is set to false in [InitFurnace](https://github.com/NREL/EnergyPlus/blob/272b4f052b19d426a0edbb4ff570d654cb6eba5b/src/EnergyPlus/Furnaces.cc#L5308-L5320)

Reply:
Not sure if there may be other places that need this - perhaps for the drawthrough logic path?

Reply:
Well maybe that's a mistake? Or maybe the supplemental picks up all the sensible load? I'm fairly positive that the furnace can meet a heating load and dehumidification load at the same time.

Reply:
So for a heating load, MoistureLoad would be true and HeatingLoad would be false. Then the supplemental would pick up the sensible load. I don't recall it that way.

Reply:
If there's a non-heatpump heating coil, then yes, the furnace can do the work, but if it's a heat pump, it can only cool or heat, it can't do both. That's why this logic happens only ` if (Furnace(FurnaceNum).FurnaceType_Num != UnitarySys_HeatCool)`. And the changes here are only in the varspd heat pump section.

Reply:
Oh yeah, forgot about the DX part. So yes, supplemental will pick up the sensible load.